src/monika/SecondMaxNum.java

```diff
@@ -0,0 +1,30 @@
+
+package monika;
+/*Assignment - 29 :
+
+Find second maximum from array without sorting the array
+*/
+
+public class SecondMaxNum {
+
+	void MaxSecondNum(int arr[]) {
+		int first = 0;
+		int second = 0;
+		for (int i = 0; i < arr.length; i++) {
+			if (arr[i] > first) {
+				second = first;
+				first = arr[i];
+			} else if (arr[i] > second) {
```

Comment:
you should one extra condition for max number is like

else if (input[index] > secondMax && input[index] != max)